README.md

```diff
@@ -358,3 +355,13 @@ spring stop # if spring is already running
 export SPRING_LOG=/tmp/spring.log
 spring rake -T
 ```
+
+## Contributing to Spring
+
+Spring is work of many contributors. You're encouraged to submit pull requests, propose
+features and discuss issues.
```

Comment:
how about:
`See [CONTRIBUTING](CONTRIBUTING.md) for pull request instructions, use [issues](https://github.com/rails/spring/issues) for feature requests, discuss issues on [stackoverflow](http://stackoverflow.com/questions/tagged/rails-spring).`